p2p/simulations/network_test.go

```diff
@@ -245,6 +245,37 @@ OUTER_TWO:
 			t.Fatalf("Superfluous conn found %v -> %v", ev.Conn.One, ev.Conn.Other)
 		}
 	}
+
+	// This test validates if all connections from the snapshot
+	// are created in the network.
```

Comment:
Actually this description doesn't distinguish it enough from the other, which kind of does this aswell?

We should tell explicitly it's about not return before all is loaded.